src/components/FullscreenLoadingIndicator.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {ActivityIndicator, StyleProp, StyleSheet, View, ViewStyle} from 'react-native';
+import styles from '../styles/styles';
+import themeColors from '../styles/themes/default';
+
+type FullScreenLoadingIndicatorProps = {
+    style?: StyleProp<ViewStyle>;
+};
+
+function FullScreenLoadingIndicator({style}: FullScreenLoadingIndicatorProps) {
+    return (
+        <View style={[StyleSheet.absoluteFillObject, styles.fullScreenLoading, style]}>
```

Comment:
Hm, so passing a `style` array here just works? Was the old logic unnecessary?

```js
const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
```

Reply:
@MaciejSWM Let's ensure this change was intended

Reply:
Turns out it works just fine. It's not only correct Typescript, but it also works when I test it with all the combinations of `style`:
http://localhost:8082/settings/profile/display-name
In `DisplayNamePage.js` (I renamed it to `.tsx` to highlight any errors, but none showed up):
```
{!props.isLoadingApp ? (
  <>
      <View style={{ height: '20%' }}>
          {/* No style */}
          <FullScreenLoadingIndicator />
      </View>
      <View style={{ height: '20%' }}>
          {/* Empty array style */}
          <FullScreenLoadingIndicator style={[]} />
      </View>
      <View style={{ height: '20%' }}>
          {/* Just on style */}
          <FullScreenLoadingIndicator style={styles.backgroundPurple} />
      </View>
      <View style={{ height: '20%' }}>
          {/* Array of styles */}
          <FullScreenLoadingIndicator style={[styles.backgroundPurple, styles.backgroundGold]} />
      </View>
  </>
) : (
```
<img width="441" alt="image" src="https://github.com/Expensify/App/assets/85628901/890279bd-6f98-410c-9da1-460e12c75f81">

Reply:
Thanks for verifying this!